src/components/common/form/AcceptPrivacyPolicyField.tsx

```diff
@@ -17,7 +17,7 @@ export default function AcceptPrivacyPolicyField({ name }: AcceptGDPRFieldProps)
       label={
         <Typography variant="body2">
           {t('validation:informed-agree-with')}{' '}
-          <ExternalLink href={routes.privacyPolicy}>{t('validation:gdpr')}</ExternalLink>
+          <ExternalLink href={routes.privacyPolicy}>{t('validation:gdpr')}</ExternalLink>.
```

Comment:
Is there a reason why the full-stop is not part of the translated string?

Reply:
If I insert it as a part of the string it is included as a link in blue. This is why I thought it is better to be this way.